cmd/katalyst-controller/app/options/spd.go

```diff
@@ -41,6 +58,19 @@ func NewSPDOptions() *SPDOptions {
 		ResyncPeriod:    time.Second * 30,
 		EnableCNCCache:  true,
 		BaselinePercent: map[string]int64{},
+
+		ResourcePortraitIndicatorPluginOptions: ResourcePortraitIndicatorPluginOptions{
```

Comment:
The initialization of `ResourcePortraitIndicatorPluginOptions` can also be streamlined using a new function similar to `NewSPDOptions`, and the methods `ApplyTo` and `AddFlags` should be implemented accordingly.